packages/ses/NEWS.md

```diff
@@ -1,9 +1,14 @@
 User-visible changes in `ses`:
 
-# Unreleased
+# Next version
+
+- Exports a `StrictModuleDescriptor` type that consists only of the
+  `NamespaceModuleDescriptor` and `SourceModuleDescriptor` that are mutually
+  supported by SES and XS.
```

Comment:
Yup. PR is clearly more than just `test:`.

Reply:
Thanks. The left of the PR description has now caught up with the right of the PR description. The tags on individual commits also capture the conventions.